test/controllers/api/v2/content_views_controller_test.rb

```diff
@@ -170,6 +171,21 @@ def test_update_components
       assert_template 'katello/api/v2/common/update'
     end
 
+    def test_duplicate_component_error_message
+      view = create(:katello_content_view)
+      view_version = create(:katello_content_view_version, :content_view => view)
+
+      composite = create(:katello_content_view, :composite)
+      ContentViewComponent.create!(:composite_content_view => composite,
+                                   :content_view_version => view_version, :latest => false)
+
+      view_version2 = create(:katello_content_view_version, :content_view => view)
+
+      put :update, params: { id: composite.id, content_view: { component_ids: [view_version.id, view_version2.id] } }
+      display_message = JSON.parse(response.body)['displayMessage']
+      assert display_message.include?("Another component already includes content view with ID")
```

Comment:
Let's change this include? to match the error message precisely - it'll make for a better test since we'll be sure nothing else made its way into the entire message